### PR TITLE
Adding a last reverted example with the new error message

### PR DESCRIPTION
**Pull request by liav-certora**
Based on the work done in this PR: https://github.com/Certora/EVMVerifier/pull/6104

### CODE REVIEW

File: CVLByExample/LastReverted/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Last Reverted
+
+Sometimes in our verification we'll want to take note of whether a call was reverted.
+In this example we have a minimal implementation of ERC20 in `ERC20.sol` and we want to verify the conditions that makes the `transfer` and `approve` functions revert. In other words we want to verify that these functions will revert if and only if certain conditions are met and never in any other case (maybe excluding some invalid states). We can achieve this by using `@withrevert` and `lastReverted`.
+
+Normally when a call reverts the prover will ignore the path the led to this call. We can signal the prover to not do that by calling the function while adding `@withrevert`.
```
Comment (yoav-el-certora) [inline CVLByExample/LastReverted/README.md:6]: Normally when a call reverts, the prover will ignore the path that led to this call

File: CVLByExample/LastReverted/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Last Reverted
+
+Sometimes in our verification we'll want to take note of whether a call was reverted.
```
Comment (nd-certora) [inline CVLByExample/LastReverted/README.md:3]:
```suggestion
Sometimes in our verification we'll want to take note of whether a call was reverted.
```
Some properties are required to reason about revert cases

File: CVLByExample/LastReverted/README.md (hunk 1 above)
Comment (nd-certora) [reply CVLByExample/LastReverted/README.md:6]: As default, the Prover ignores reverting paths.

File: CVLByExample/LastReverted/RevertingConditionsForgotWithRevert.spec [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    require e.msg.value == 0;
+    require amount + balanceOf(recipient) <= max_uint256;
+    
+    bool senderIsZero = e.msg.sender == 0;
+    bool recipientIsZero = recipient == 0;
+    bool notEnoughFunds = balanceOf(e.msg.sender) < amount;
+    
+    transfer(e, recipient, amount);
+
+    assert lastReverted <=> senderIsZero || recipientIsZero || notEnoughFunds;
```
Comment (nd-certora) [inline CVLByExample/LastReverted/RevertingConditionsForgotWithRevert.spec:21]: lets make this more interesting and common:

```
transfer@withrevert(e, recipient, amount);

assert balanceOf(e.msg.sender) > balanceBefore - amount =>  lastReverted

**Review by nd-certora — APPROVED** [review]

File: CVLByExample/LastReverted/RevertingConditionsForgotWithRevert.spec (hunk 3 above)
Comment (liav-certora) [reply CVLByExample/LastReverted/RevertingConditionsForgotWithRevert.spec:21]: Nice one Nurit, this doesn't work: https://prover.certora.com/output/15800/4e4bcbd0c5384d0ab0e7109748e5cb1f?anonymousKey=7f7323ca7416e69a40bafc323b10106d5682356a

**Review by nd-certora — APPROVED** [review]

**Pull request merged**